content/en/tracing/setup/_index.md

```diff
@@ -1,33 +1,21 @@
 ---
-title: Application Tracing Setup
+title: Configure your application to collect traces
 kind: documentation
 aliases:
     - /tracing/languages
     - /tracing/proxies
 disable_toc: true
 ---
 
-After you have [installed the Datadog Agent][1], [enabled trace collection][2], and [configured your environment][3], begin tracing by instrumenting your application.
+After you have [installed the Datadog Agent][1], and [enabled trace collection][2], then configure your application to send traces using one of the libraries locally, or in [a container][3].
```

Comment:
```suggestion
After you have [enabled trace collection][2], configure your application to send traces using one of the following official Datadog tracing libraries:
```

Reply:
removing the Agent reference since it's only one of several ways to collect traces.